test/fixtures/bpmn/non-regression/gateways.bpmn

```diff
@@ -6,6 +6,8 @@
     <bpmn:inclusiveGateway id="Gateway_1xb9nl5" />
     <bpmn:complexGateway id="Gateway_1qm3uqe" />
     <bpmn:eventBasedGateway id="Gateway_1n1md3j" />
+    <bpmn:eventBasedGateway id="Gateway_1fcfrqc" instantiate="true" />
+    <bpmn:eventBasedGateway id="Gateway_0i6s6lx" instantiate="true" eventGatewayType="Parallel" />
```

Comment:
Missing same test with Exclusive

Reply:
Here we want to test the way we render the various flavours of the shape.
Putting `instantiate=true` and `eventGatewayType=Exclusive` will produce the same result as with `instantiate=true` alone.

This is something that is managed at parsing time: the produced BpmnModel is the same, so the mxgraph model and the rendering will then be the same as well.

This use case is currently missing in `BpmnJsonParser.flowNode.test.ts`, I will add a test there to show that we get the same BpmnModel result.

Reply:
👍